src/IconView.vala

```diff
@@ -103,6 +104,26 @@ public class IconView : Gtk.Box {
         source_view.add_css_class (Granite.STYLE_CLASS_CARD);
         source_view.add_css_class (Granite.STYLE_CLASS_ROUNDED);
 
+        copy_button = new Gtk.Button.with_label (_("Copy")) {
+            valign = START,
+            halign = END,
+            visible = false,
```

Comment:
Instead of using `visible` property, we can add a Gtk.Revealer to get a nice fade in animation. Just make sure you set `overflow` visible on the revealer to not clip shadows